examples/lambda/barendregt/chap2Script.sml

```diff
@@ -106,6 +106,7 @@ val lameq_weaken_cong = store_thm(
   METIS_TAC [lameq_rules]);
 
 Theorem lameq_SYM = List.nth(CONJUNCTS lameq_rules, 2)
```

Comment:
If you use the labelling option in the Inductive definition of `lameq` you should be able to get these for free:
```
...
[~SYM:]
(!M N. M == N ==> N == M) /\
[~TRANS:]
(!M N P. M == N /\ N == P ==> M == P) /\ ...
```
You will need to replace the call to `xHol_reln` with `Inductive lameq:`